app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ Optional ] Kindly I recommend you  use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new comment has been saved to database, then make the update_comments_counter method private Just like this :- 

  # app/models/comment.rb
```
  after_save :update_comments_counter

  private

  def update_comments_counter
    post.increment!(:comments_counter)
  end
```

Reply:
Hi @Abenezer-Tilahun 

Just wanted to drop a quick note to say thank you for reviewing my code. Your feedback and suggestions have been incredibly valuable, and I truly appreciate your time and expertise. I would like to connect with you on [GitHub ](https://github.com/brainskev) & [LinkedIn](https://www.linkedin.com/in/kelvinkaviku/) to stay in touch and potentially collaborate on future projects. Your contributions have been invaluable, and I believe we can learn a lot from each other.

Looking forward to connecting!

Best regards,
Kelvin Kaviku 
@brainskev